### PR TITLE
Url fixed.  Sends cookie from backend storing user information

### PR DESCRIPTION
**Pull request by stevenshuhyow**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "pg": "4.3.0",
     "pg-hstore": "2.3.2",
     "react": "0.13.3",
+    "react-cookie": "^0.2.2",
```
Comment (dmsakamoto) [inline package.json]: Make stable edition - remove ^

File: client/public/js/stores/AppStore.js [hunk 2]
```diff
@@ -60,11 +61,11 @@ AppDispatcher.register(function(payload) {
       break;
 
     case AppConstants.REMOVE_CURRENT_USER:
-      // console.log('removing user inside of appstore.js');
+      console.log('removing cookie user inside of appstore.js');
```
Comment (ekong2) [inline client/public/js/stores/AppStore.js]: you're not gonna comment this out? :)

File: server/auth/authRoutes.js [hunk 3]
```diff
@@ -12,11 +12,15 @@ module.exports = function(app){
...
-        res.redirect(
-          '/?' + querystring.stringify({user: user.username}) +
-          '&' + querystring.stringify({id: user.table_id})+
-          '&' + querystring.stringify({token: token})
-        );
+        res.cookie('username', user.username);
+        res.cookie('id', user.table_id);
+        res.redirect('/events');
+
+        // res.redirect(
```
Comment (ekong2) [inline server/auth/authRoutes.js:19]: can you remove this? deprecated code

**Comment by kevhuang** [comment]
:+1:

**Comment by ekong2** [comment]
:+1:  here's your thumbs

**Pull request merged**